smoke-tests/src/test/java/org/hypertrace/agent/smoketest/SpringBootSmokeTest.java

```diff
@@ -79,45 +79,33 @@ public void get() throws IOException, InterruptedException {
             .getMainAttributes()
             .get(Attributes.Name.IMPLEMENTATION_VERSION);
 
-    Assertions.assertEquals(
-        ResourceAttributes.CONTAINER_ID.getKey(),
-        traces.get(0).getResourceSpans(0).getResource().getAttributes(0).getKey());
     Assertions.assertEquals(
         ResourceAttributes.SERVICE_NAME.getKey(),
-        traces.get(0).getResourceSpans(0).getResource().getAttributes(9).getKey());
-    Assertions.assertEquals(
-        ResourceAttributes.TELEMETRY_AUTO_VERSION.getKey(),
-        traces.get(0).getResourceSpans(0).getResource().getAttributes(10).getKey());
-    Assertions.assertEquals(
-        ResourceAttributes.TELEMETRY_SDK_LANGUAGE.getKey(),
-        traces.get(0).getResourceSpans(0).getResource().getAttributes(11).getKey());
-    Assertions.assertEquals(
-        ResourceAttributes.TELEMETRY_SDK_NAME.getKey(),
-        traces.get(0).getResourceSpans(0).getResource().getAttributes(12).getKey());
+        traces.get(0).getResourceSpans(0).getResource().getAttributes(0).getKey());
```

Comment:
Rather than revert these tests, can we configure the tests to run with otlp?

Reply:
Sure, we can use OTLP in config while running these tests.